WordPress/gutenberg
# Navigation: make additional top margin specific to open modal

## What?

In https://github.com/WordPress/gutenberg/pull/41664, we added a top margin to the mobile menu when the admin bar is visible (or when the user is logged in). I didn't realise this also affects the nav items when the mobile menu is closed, which was unintentional. This change only applies the top margin when the mobile menu is open.

## Why?

Fix a spacing bug introduced in https://github.com/WordPress/gutenberg/pull/41664.

## How?

By making the selector more specific to an open modal.

## Testing Instructions




1. Activate Twenty Twenty-Two
2. Make sure there is a navigation block in the header with the Overlay Menu set to Mobile
```



```
3. Make sure you're logged in
4. Ensure the nav items do not have a top margin when the mobile menu is closed (i.e. the screen width is large enough to show the full menu)

## Screenshots or screencast 

Before:
<img width="272" alt="image" src="https://user-images.githubusercontent.com/1645628/175552269-a6836c72-3e85-4206-b06a-96e80dddaffb.png">

After:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/1645628/175552051-ad3448df-c3e7-40e8-a03b-8fb737a19547.png">
